samples/bluetooth/channel_sounding_ras_initiator/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _channel_sounding_ras_initiator:
+
+Bluetooth: Channel Sounding Initiator with Ranging Requestor
+############################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to use the ranging service to request ranging data from a server.
+It also provides a basic distance estimation algorithm to show how channel sounding can be used to estimate distance between two devices.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also requires a device running a Channel Sounding Reflector with Ranging Responder to connect to, such as the :ref:`channel_sounding_ras_reflector` sample.
+
+Overview
+********
+
+The sample demonstrates a basic Bluetooth® Low Energy Central role functionality that acts as a GATT Ranging Requestor client and configures the Channel Sounding initiator role.
+Regular Channel Sounding procedures are set up, local subevent data is stored and peer ranging data is fetched.
+
+A basic distance estimation algorithm is included in the sample.
+The mathematical representations described in `Distance estimation based on phase and amplitude information`_ and `Distance estimation based on RTT packets`_ are used as the basis for this algorithm.
+
+User interface
+**************
+
+The sample does not require user input and will scan for a device advertising with the GATT Ranging Service UUID.
+The first LED on the development kit will be lit when a connection has been established.
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/bluetooth/channel_sounding_ras_initiator`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+After programming the sample to your development kit, you can test it by connecting to another device programmed with a Channel Sounding Reflector role with Ranging Responder.
+
+1. |connect_terminal_specific|
+#. Reset both kits.
+#. Program the other kit with the Channel Sounding Reflector with Ranging Responder sample.
```

Comment:
```suggestion
#. Program the other kit with the :ref:`channel_sounding_ras_reflector` sample.
```

Reply:
Realized this line should have been deleted as per https://github.com/nrfconnect/sdk-nrf/pull/18921#discussion_r1853449957